src/mockServiceWorker.js

```diff
@@ -53,6 +54,18 @@ self.addEventListener('message', async function (event) {
       break
     }
 
+    case 'SHARED_MOCK_ACTIVATE': {
```

Comment:
This case behaves the same as `MOCK_ACTIVATE` with the exception of setting the `sharedClientId` global variable. Once more I'm wondering if we can do that within `MOCK_ACTIVATE` and skip the introduction of a new event for that only purpose. We can still transfer additional data in `event.data`, so even `MOCK_ACTIVATE` event can contain something like `event.data.sharedClientId` to determine the need of setting `sharedClientId`.